test/cpp/naming/resolver_component_test.cc

```diff
@@ -49,11 +49,20 @@ extern "C" {
 using std::vector;
 using testing::UnorderedElementsAreArray;
 
-DEFINE_string(target_name, "", "");
-DEFINE_string(expected_addrs, "", "");
-DEFINE_string(expected_chosen_service_config, "", "");
-DEFINE_string(local_dns_server_address, "", "");
-DEFINE_string(expected_lb_policy, "", "");
+DEFINE_string(target_name, "", "Target name to resolve.");
+DEFINE_string(
+    expected_addrs, "",
+    "Comma-separated list of expected '(<ip>:<port>),...' addresses of "
```

Comment:
This doesn't say how the is_balancer bit is encoded.

Also, the parens seem a little cumbersome.  I was thinking of something more like "address0,is_balancer0;address1,is_balancer1;..." -- i.e., use comma between the two elements of each pair and semicolon between the pairs.  Or perhaps "address0=is_balancer0,address1=is_balancer1,...".